package.json

```diff
@@ -21,7 +21,6 @@
     "@types/node": "^10.17.27",
     "@typescript-eslint/eslint-plugin": "^4.9.1",
     "@typescript-eslint/parser": "^4.9.1",
-    "aws-cdk": "1.71.0",
```

Comment:
I think this still might be required as a dev dependency. We need to be able to run the `cdk` command as a developer of this project.